changelog.md

```diff
@@ -1,3 +1,14 @@
+19 March 2018, v1.4.0
+---------------------
+
+- update nunit dependency to v3.10.0
+- update NUnit3TestAdapter dependency to v3.10.0
+- update Microsoft.NET.Test.Sdk dependency to v15.6.1
+- add new `--framework` supported parameters:
+    + .NET Framework versions (net40, net45, net451, net452, net46, net461, net462, net47, net471)
```

Comment:
It looks like the net40 target is listed here, but not actually supported? Unless I'm misreading these commits?

(FWIW - I'd love it to be added if there's not additional cost, I work in .NET 4.0 on a day-to-day basis. 😄)

Reply:
Thanks for feedback! Yes, you right about net40. I'm have returned back it now)

Reply:
Awesome - thanks!